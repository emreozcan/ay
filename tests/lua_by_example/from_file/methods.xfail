We do not take the __index metavalue into account in lookups yet.
