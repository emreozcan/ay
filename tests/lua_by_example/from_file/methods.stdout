shared:	string is shared	unique: string is unique
shared:	string is shared	unique: string is unique

shared:	new shared state	unique: string is unique
shared:	new shared state	unique: string is unique

shared:	overwrite first	unique: overwrite first
shared:	overwrite second	unique: overwrite second

shared:	overwrite first	unique: overwrite first
shared:	overwrite second	unique: overwrite second
shared:	only in inst. c	unique: string is unique
